examples/basics/scene/volume.py

```diff
@@ -10,11 +10,12 @@
 
 Controls:
 
-* 1 - toggle camera between first person (fly) and regular 3D (turntable)
-* 2 - toggle between mip and iso render styles
-* 3 - toggle between stent-CT / brain-MRI image
-* 4 - toggle between colormaps
-* 0 - reset cameras
+* 1  - toggle camera between first person (fly) and regular 3D (turntable)
+* 2  - toggle between volume rendering methods
+* 3  - toggle between stent-CT / brain-MRI image
+* 4  - toggle between colormaps
+* 0  - reset cameras
+* [] - decrease/increase isosurface threshold
```

Comment:
Minor issues:

* camera reset doesn't seem to do anything for me for TurntableCamera, and for Arcball it only resets the distance (not rotation)
* isosurface threshold can be set negative (should probably be limited here >= 0)
* scroll wheel with FlyCamera doesn't seem to work:
```
  File "/Users/larsoner/custombuilds/vispy/vispy/scene/cameras/cameras.py", line 1567, in viewbox_mouse_event
    PerspectiveCamera.viewbox_mouse_event(self, event)
  File "/Users/larsoner/custombuilds/vispy/vispy/scene/cameras/cameras.py", line 764, in viewbox_mouse_event
    if self._distance is not None:
```
These issues seem to also be in `master`, so I can open a new issue for those if you don't want to fix them here.